doc/host-applications.rst

```diff
@@ -315,5 +315,5 @@ Our examples always passed the full input at once. You don't need to
 do that, Spicy's parsers can process input incrementally as it comes
 in, and return back to the caller to retrieve more. See the source of
 `spicy::Driver::processInput()
-<https://github.com/zeek/spicy/blob/master/spicy/src/rt/driver.cc#L121>`_
+<https://github.com/zeek/spicy/blob/4e3fcb80d19aaa2b9b205134b8e2453348dbaf50/spicy/runtime/src/driver.cc#L144>`_
```

Comment:
This risks showing outdated code at some point in the future. Not sure what's better: that, or showing wrong line numbers. Hard to catch either way.

Reply:
I changed this to now link to only the file, not the line number.